source/transactions/tests/README.rst

```diff
@@ -77,6 +77,11 @@ control the fail point's behavior. ``failCommand`` supports the following
   blocked for. Required when blockConnection is true.
   `New in mongod 4.3.4 <https://jira.mongodb.org/browse/SERVER-41070>`_.
 
+Speeding Up Tests
+=================
+
+Drivers should set the default heartbeatFrequencyMS to 5ms in order to take into account the latest changes regarding streaming protocol. If a test has an explicit heartbeatFrequencyMS value, drivers should use the explicit value.
```

Comment:
Reading this alongside the existing paragraph on tweaking `minHeartbeatFrequencyMS` in Retryable Reads, how would drivers be able to tweak `heartbeatFrequencyMS` if `minHeartbeatFrequencyMS` is already [fixed at 500ms](https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-monitoring.rst#minheartbeatfrequencyms)?

I think this text should state that drivers manually lower `minHeartbeatFrequencyMS` if possible _and_ reduce `heartbeatFrequencyMS` to that same value -- with an exception if the test already has an explicit `heartbeatFrequencyMS` value. You can suggest 5ms for both if you wish.

That said, I'm not sure why we should bother adding anything to Transactions if we already have a section in the Retryable Reads spec. Perhaps just have everything in this PR refer to the Retryable Read section instead (and modify it as you like).

Reply:
> I think this text should state that drivers manually lower minHeartbeatFrequencyMS if possible and reduce heartbeatFrequencyMS to that same value -- with an exception if the test already has an explicit heartbeatFrequencyMS value. You can suggest 5ms for both if you wish.

Agreed. @DmitryLukyanov can you confirm that reducing both these values to 5ms does not cause any tests to fail?

Reply:
@ShaneHarvey , I can confirm this: https://evergreen.mongodb.com/version/5f5ffde90ae6064926c283ff. 

`heartbeatFrequencyMS` has been reduced during the last month, for `minHeartbeatFrequencyMS`, I needed to make a code change. So, I'm not sure how strong required reducing minHeartbeatFrequencyMS, I would say not really

Reply:
Thanks for testing that. I second Jermey's request that we mention Drivers may reduce `minHeartbeatFrequencyMS`as well (if possible in the language). Note that we will not require drivers to reduce `minHeartbeatFrequencyMS`, nor force drivers to make `minHeartbeatFrequencyMS` configurable by users (because it shouldn't be). We can just suggest it as a possible way to speed up slow tests.

Reply:
Please make the changes in my earlier comment (https://github.com/mongodb/specifications/pull/820/files#r458262676) for retryable reads and change this to refer to that spec. I think the content can match what I suggested for retryable writes, which is just a reference to the retryable reads section.

Reply:
done